Lib/unittest/test/testmock/testpatch.py

```diff
@@ -2,19 +2,19 @@
 # E-mail: fuzzyman AT voidspace DOT org DOT uk
 # http://www.voidspace.org.uk/python/mock/
 
+import contextlib
 import os
 import sys
-
 import unittest
-from unittest.test.testmock import support
-from unittest.test.testmock.support import SomeClass, is_instance
 
+from test.support import swap_attr
 from unittest.mock import (
     NonCallableMock, CallableMixin, sentinel,
     MagicMock, Mock, NonCallableMagicMock, patch, _patch,
     DEFAULT, call, _get_target
 )
-
+from unittest.test.testmock import support
+from unittest.test.testmock.support import SomeClass, is_instance
```

Comment:
Could you avoid rearranging the imports? It makes it hard to see what is being changed.

Reply:
Sorry. I just fixed it in a recent commit.